### PR TITLE
Update CMakeLists.txt as needed. 

### PR DESCRIPTION
**Pull request by eflumerf**
Add script to show when files have been added or removed

### CODE REVIEW

**Comment by eflumerf** [comment]
Commit [1560c76](https://github.com/Mu2e/Offline/pull/1221/commits/1560c76110e7f28367fb02993706460f1c1c4bd3) was tagged with "d11_01_00" and UPS products were deployed on the mu2edaq machines based on this tag for the DAQ Global Run 1.

**Comment by kutschke** [comment]
@FNALbuild run build test

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,4 @@ tags
 .clangd/*
 compile_commands.json
 spack-*
+Offline
```
Comment (kutschke) [inline .gitignore:47]: I am curious why you need this?  Is it because of the symlink you need to add?

**Review by kutschke — APPROVED** [review]
I see that you removed a lot of configure_file COPYONLY cmake commands.  Are these files now found in the source tree ( or a copy of it?).

**Comment by kutschke** [comment]
@eflumerf I added a few questions that I would like answered - for my own background information.  I am merging now.

File: .gitignore (hunk 1 above)
Comment (eflumerf) [reply .gitignore:47]: Yes

**Comment by eflumerf** [comment]
> I see that you removed a lot of configure_file COPYONLY cmake commands. Are these files now found in the source tree ( or a copy of it?).

configure_file only puts things into the build tree. The install(DIRECTORY) command I replaced them with puts them into the appropriate place in the install tree (whether UPS product or Spack installation), and it also copies the files recursively so that we don't need to maintain a configure_file command for each data file.

**Pull request merged**